platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/address/AddressBookUtils.java

```diff
@@ -93,6 +93,13 @@ public static Address parseAddressConfigText(
         } catch (NumberFormatException e) {
             throw new ParseException("Cannot parse ip port from '" + parts[7] + "'", 7);
         }
+        final String memoToUse;
+        if (parts.length == 9) {
```

Comment:
Perhaps it should be noted in the javadoc that the `memo` parameter is ignored if a memo already exists in the line